src/libaktualizr-c/libaktualizr-c.cc

```diff
@@ -135,3 +137,70 @@ int Aktualizr_install_target(Aktualizr *a, Target *t) {
   return 0;
 }
 
+int Aktualizr_send_manifest(Aktualizr *a, const char *manifest) {
+  try {
+    Json::Value custom = Utils::parseJSON(manifest);
+    bool r = a->SendManifest(custom).get();
+    return r ? 0 : -1;
+  } catch (const std::exception &e) {
+    std::cerr << "Aktualizr_send_manifest exception: " << e.what() << std::endl;
+    return -1;
+  }
+}
+
+int Aktualizr_send_device_data(Aktualizr *a) {
+  try {
+    a->SendDeviceData();
+    return 0;
+  } catch (const std::exception &e) {
+    std::cerr << "Aktualizr_send_device_data exception: " << e.what() << std::endl;
+    return -1;
+  }
+}
+
+void *Aktualizr_open_stored_target(Aktualizr *a, const char *filename, const char *content) {
```

Comment:
The C api already defines a `Target` type, can we use it here as a parameter instead of re-parsing Json? Also the `content` parameter name is quite non-descriptive.

Reply:
Fixed.